Tests/DiligentCoreAPITest/src/TessellationTest.cpp

```diff
@@ -152,6 +152,7 @@ TEST(TessellationTest, DrawQuad)
 
     ShaderCreateInfo ShaderCI;
     ShaderCI.SourceLanguage             = SHADER_SOURCE_LANGUAGE_HLSL;
+    ShaderCI.ShaderCompiler             = pEnv->GetDefaultCompiler(ShaderCI.SourceLanguage);
```

Comment:
This seems to be working just fine with DEFAULT compiler, so why do we need this?